packages/google-pay-integration/src/google-pay-paypal-commerce/types.ts

```diff
@@ -13,10 +13,15 @@ export interface PayPalSDK {
             orderId: string;
             paymentMethodData: ConfirmOrderData;
         }) => Promise<{ status: string }>;
-        initiatePayerAction: () => void;
+        initiatePayerAction: (arg0: { orderId: string }) => Promise<void>;
```

Comment:
Is it possible to change `arg0` to something more meaningful?

Reply:
Done